utility.py

```diff
@@ -183,6 +183,10 @@ def setup_logging(args: dict):
     logger.addHandler(log_file_handler)
 
 
+def smoothstep(t: float):
+    return t * t * (3 - 2 * t)
```

Comment:
Are these values; `3` and `2` specific for our noise-values? If so, maybe a comment on it is appropriate.

Reply:
Nope. They are specific to the smoothstep function, and are the exact numbers needed to make the graph go through origin (0, 0) and the point (1, 1).